fendermint/ipc/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+//! Interfacing with IPC, provides utility functions
+
+mod agent;
+mod cache;
+mod message;
+pub mod parent;
+pub mod pof;
+
+use crate::pof::IPCParentFinality;
+use async_trait::async_trait;
+use ipc_sdk::cross::CrossMsg;
+use ipc_sdk::ValidatorSet;
+pub use message::IPCMessage;
+
+#[derive(Debug, Clone)]
+pub struct Config {
+    /// The number of blocks to delay reporting when creating the pof
+    chain_head_delay: BlockHeight,
+    /// The lower bound for the chain head height in parent view
+    chain_head_lower_bound: BlockHeight,
```

Comment:
It would be helpful to explain here whether this is an _absolute_ value or a _relative_ one compared to the latest height.

Reply:
https://github.com/consensus-shipyard/fendermint/pull/181/files#diff-088fbb2a5e5eeef38c5648a53f47e659eecd568fe77bb93e6b262cfa08801b42R185 

suggests it's a delta, but then what's the difference between this and the "delay" field? :thinking: